### DISCUSSION TITLE
Home assistant integration access from pi on same network.

### DISCUSSION THREAD

**Discussion by Scales82** [htpcBeginner/docker-traefik#289] [Support]
Hi,
Love this project, have been using it as a basis for years on my home server. I am probably a little behind on the compose file though.

Anyway.... to my problem.

I recently setup home assistant OS on a pi4 on my home network. I am having no end of drama trying to get the integrations (Sonarr, Radarr, SAB, tautulli etc). to talk to my HTPC server 'box' which is based on this git. I am using the oauth method and I am kind of assuming it is related to that, or maybe the bypass rules, but I honestly have no idea why its not working.
I just cant get any of the integrations to talk to my box to get information from it. I need some help. Can anyone assist where I should be looking?

**Comment by SimpleHomelab** [comment]
Please hop on to our discord for faster help. We need more details.
